lib/tasks/data_fixes/update_related_urls.rake

```diff
@@ -0,0 +1,23 @@
+require "#{Rails.root}/lib/scihist_digicoll/task_helpers/related_url_updater"
```

Comment:
`./lib` should already be on the ruby "load path", so you should be able to simply do:

    require `scihist_digicoll/task_helpers/related_url_updater`

Which would be preferable. If it doesn't work, I'll be curious!